### PR TITLE
Additional Crons docs improvements

### PR DESCRIPTION
**Pull request by gaprl**
- fixes incorrect example for heartbeat request
- adds new configuration instructions for Crons CLI

### CODE REVIEW

File: src/docs/product/crons/getting-started/cli.mdx [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
 ```bash {tabTitle: CLI}
-sentry-cli monitors run <monitor_id> -- <command> <args>
+sentry-cli monitors run <monitor_slug> -- <command> <args>
 ```
 
 Usage examples:
 
 ```bash {tabTitle: Python}
-sentry-cli monitors run 63261d96-2027-472a-a8bb-9895dbebd793 -- python path/to/file.py
+sentry-cli monitors run my_monitor_slug -- python path/to/file.py
```
Comment (evanpurkhiser) [inline src/docs/product/crons/getting-started/cli.mdx:51]:
```suggestion
sentry-cli monitors run my-monitor-slug -- python path/to/file.py
```

File: src/docs/product/crons/getting-started/cli.mdx [hunk 2]
```diff
@@ -1,35 +1,56 @@
...
 ```
 
 ```bash {tabTitle: Node.JS}
-sentry-cli monitors run 63261d96-2027-472a-a8bb-9895dbebd793 -- node path/to/file.js
+sentry-cli monitors run my_monitor_slug -- node path/to/file.js
```
Comment (evanpurkhiser) [inline src/docs/product/crons/getting-started/cli.mdx:55]:
```suggestion
sentry-cli monitors run my-monitor-slug -- node path/to/file.js
```

We support `_` but prefer `-`

**Review by evanpurkhiser — COMMENTED** [review]
This reads good to me. Will let @lizokm approve

File: src/docs/product/crons/getting-started/cli.mdx [hunk 3]
```diff
@@ -1,35 +1,56 @@
...
-draft: true
 ---
 
 <Include name="beta-alert-crons.mdx" />
 
+<Alert level="warning" title="Deprecation Notice">
+
+Starting from v2.16.0 the Sentry CLI enables monitor check-in authorization using your project's DSN. **Monitor check-in authorization through an auth token is deprecated.** See instructions below to update your CLI to use your project's DSN.
+
+</Alert>
```
Comment (lizokm) [inline src/docs/product/crons/getting-started/cli.mdx]:
```suggestion
<Alert level="warning" title="Deprecation Notice">

Starting from v2.16.0, the Sentry CLI enables monitor check-in authorization using your project's DSN. **The ability to monitor check-in authorization through an auth token is deprecated.** See instructions below to update your CLI and use your project's DSN.

</Alert>
```

File: src/docs/product/crons/getting-started/cli.mdx [hunk 4]
```diff
@@ -1,35 +1,56 @@
...
 To begin monitoring your recurring, scheduled job:
 
-- [Install](/product/cli/installation) and [configure](/product/cli/configuration) the Sentry CLI.
-    - The authorization token provided requires the `project:write` scope.
+- [Install](/product/cli/installation) the Sentry CLI (min v2.16.0).
 - [Create and configure](https://sentry.io/crons/create/) your first Monitor.
 
+## Configuration
+
+The Sentry CLI utilizes your Monitor's project DSN to authorize check-ins. To set up, you can export the `SENTRY_DSN` environment variable:
```
Comment (lizokm) [inline src/docs/product/crons/getting-started/cli.mdx:25]:
```suggestion
The Sentry CLI uses your Monitor's project DSN to authorize check-ins. To set it up, export the `SENTRY_DSN` environment variable:
```

**Review by lizokm — APPROVED** [review]
Minor tweaks.

**Comment by gaprl** [comment]
Hey @lizokm made a few more changes, do you mind taking a look? Thank you.

File: src/docs/product/crons/getting-started/index.mdx [hunk 5]
```diff
@@ -2,17 +2,25 @@
 title: Set Up
 sidebar_order: 0
 redirect_from:
-- /product/crons/alerts/
+  - /product/crons/alerts/
+description: "Get started with Sentry's Cron Monitoring, which allows you to monitor the uptime and performance of any scheduled, recurring job."
```
Comment (lizokm) [inline src/docs/product/crons/getting-started/index.mdx:6]:
```suggestion
description: "Learn how to set up Sentry's Cron Monitoring using our CLI, HTTP, and supported SDKs."
```

File: src/docs/product/crons/getting-started/index.mdx [hunk 6]
```diff
@@ -2,17 +2,25 @@
...
 ---
 
-To set up Sentry Crons, use the links for CLI, HTTP, and supported SDKs below. Crons allows you to monitor the uptime and performance of any scheduled, recurring job.
+To set up Sentry Crons, use the links below for supported SDKs or use the Sentry CLI. After set up, you'll be able to receive alerts if your job fails to run at its scheduled time or doesn't execute successfully.
```
Comment (lizokm) [inline src/docs/product/crons/getting-started/index.mdx:9]:
```suggestion
To set up Sentry Crons, use the links below for supported SDKs or the Sentry CLI. After it's been set up, you'll be able to receive alerts if your job fails to run at its scheduled time or doesn't execute successfully.
```

File: src/docs/product/crons/getting-started/index.mdx [hunk 7]
```diff
@@ -2,17 +2,25 @@
...
 - [PHP](/platforms/php/crons/)
-    - [Laravel](/platforms/php/guides/laravel/crons/)
+  - [Laravel](/platforms/php/guides/laravel/crons/)
 - [Python](/platforms/python/crons/)
-    - [Celery](/platforms/python/guides/celery/crons/)
+  - [Celery](/platforms/python/guides/celery/crons/)
+
+<Alert level="note" title="Don't see your platform?">
+
+We are are working on enabling additional platforms for Crons. Meanwhile, you can use the Sentry CLI or via HTTP to set up monitoring for your jobs.
```
Comment (lizokm) [inline src/docs/product/crons/getting-started/index.mdx:21]:
```suggestion
We're working on enabling additional platforms for Crons. In the meantime, you can use the Sentry CLI or HTTP to set up monitoring for your jobs.
```

**Review by lizokm — APPROVED** [review]
Small tweaks :)

**Pull request merged**